proposals/text/7462-rag-evaluation.md

```diff
@@ -87,30 +84,18 @@ Example output
 A detailed evaluation report that provides the scores of all available metrics for all queries or a subset of queries.
 
 ```python
-def get_detailed_scores(query_ids: Union[List[str], str] = "all"):
+def individual_detailed_score_report(queries: Union[List[str], str] = "all"):
 ```
 
 Example output
 
 ```bash
-data  = {
-    "query_id": ["53c3b3e6, 225f87f7, 8ac473ec, 97d284ca"],
-    "reciprocal_rank": [0.378064, 0.534964, 0.216058, 0.778642],
-    "single_hit": [1, 1, 0, 1],
-    "multi_hit": [0.706125, 0.454976, 0.445512, 0.250522],
-    "context_relevance": [0.805466, 0.410251, 0.750070, 0.361332],
-    "faithfulness": [0.135581, 0.695974, 0.749861, 0.041999],
-    "semantic_answer_similarity": [0.971241, 0.159320, 0.019722, 1],
-    "aggregated_score":
-      {
-          'Reciprocal Rank': 0.448,
-          'Single Hit': 0.5,
-          'Multi Hit': 0.540,
-          'Context Relevance': 0.537,
-          'Faithfulness': 0.452,
-          'Semantic Answer Similarity': 0.478
-      }
-}
+| query_id | reciprocal_rank | single_hit | multi_hit | context_relevance | faithfulness | semantic_answer_similarity |
```

Comment:
Could we remove `query_id` here and add the columns from the inputs, i.e., `question`, `context`, `answer`. No need to fill each row with actual text (`...` should be enough).

Reply:
ok